install_config/configuring_authentication.adoc

```diff
@@ -159,18 +159,24 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Configuring the ldap ca certificate <1>
 #openshift_master_ldap_ca=<ca text>
 # or
-#openshift_master_ldap_ca_file=<path to local ca file to use>
+#openshift_master_ldap_ca_file=<path to local ca file to use> <2>
 
 # Available variables for configuring certificates for other identity providers:
 #openshift_master_openid_ca
-#openshift_master_openid_ca_file
+#openshift_master_openid_ca_file <2>
 #openshift_master_request_header_ca
-#openshift_master_request_header_ca_file
+#openshift_master_request_header_ca_file <2>
 ----
 <1> If you specify your CA certificate location in the
 `openshift_master_identity_providers` parameter, do not specify a certificate
 value in the `openshift_master_ldap_ca` parameter or path in the
 `openshift_master_ldap_ca_file` parameter.
+<2> If you specify a file on the host you run the playbook on, its contents are
+copied to the *_/etc/origin/master/<identity_provider_name>_ca.crt_* file. The
+identity provider name is `ldap`, `openid`, or `request_header` to match the
+identity provider that you configure. If you do not
+specify the CA text or the path to the local CA file, you must place the CA
+cert in this location. You cannot change this location.
```

Comment:
In LDAP `insecure` option would allow omitting `openshift_master_ldap_ca` or `openshift_master_ldap_ca_file`